python/ray/experimental/workflow/common.py

```diff
@@ -35,21 +31,38 @@ class WorkflowStatus(str, Enum):
 
 @dataclass
 class WorkflowInputs:
-    # The workflow step function body.
-    func_body: Callable
     # The object ref of the input arguments.
     args: ObjectRef
     # The hex string of object refs in the arguments.
-    object_refs: List[str]
+    object_refs: List[ObjectRef]
     # The ID of workflows in the arguments.
```

Comment:
The workflows in the arguments

Reply:
Btw, should `WorkflowData` be put here instead of Workflow? The thing that should be carried around is workflow data right?

Reply:
no, currently we need to call `execute()` of the workflow. in the future we can totally decouple them